components/actionbutton/index.css

```diff
@@ -232,6 +232,8 @@ a.spectrum-ActionButton {
 }
 
 .spectrum-ActionButton-label {
+	line-height: var(--spectrum-actionbutton-height);
+
```

Comment:
Great addition! Can we add this below the extend? I think ideally the extends appear at the top of the style block.

Reply:
ahhh yes absolutely. And I also added the test case as well.